floodlight/io/catapultApi.py

```diff
@@ -96,87 +111,123 @@ def _getAllplayerFormActivity_response(baseUrl: str, api_token: str, activity_id
     }
     endpoint = f"activities/{activity_id}/athletes/"
     response = requests.get(baseUrl + endpoint, headers=headers)
-    if response.status_code == 200:
-        return response
+    response_ok =200
+    if response.status_code == response_ok:
+        response_data = response.json()
+        
+        return response_data, endpoint
     else:
         raise APIRequestError(f"API request failed with status code {response.status_code}")
 
-def getResponse_data_json_list(baseUrl: str, api_token: str, activity_id: str) -> List[Dict]:
+def getResponse_data_dict_list(baseUrl: str, api_token: str, activity_id: str, response: List[Dict]) -> List[Dict]:

```

Comment:
As above, I think `get_response_data_dict_list()` and `base_url` would be more consistant

Reply:
done